src/main/java/profile_manager/application_business_rules/ProfileRepoRequestModel.java

```diff
@@ -0,0 +1,158 @@
+package profile_manager.application_business_rules;
+
+import profile_manager.frameworks_and_drivers.LoginScreen;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ProfileRepoRequestModel {
+
+    private final String userName;
+    private String profileName;
+    private LocalDate dob;
+    private String description;
+    private List<String> socialLinks;
+    private List<String> sensitiveWords;
+    private List<String> interests;
+    private List<String> groups;
+    private List<String> friends;
+    private List<String> blockedUsers;
+
+    private final LocalDateTime creationTime;
+
+    public ProfileRepoRequestModel(String userName,
+                                   String profileName,
+                                   LocalDate dob,
+                                   String description,
+                                   List<String> socialLinks,
+                                   List<String> sensitiveWords,
+                                   List<String> interests,
+                                   List<String> groups,
+                                   List<String> friends,
+                                   List<String> blockedUsers,
+                                   LocalDateTime creationTime) {
+
+        this.userName = userName;
+        this.profileName = profileName;
+        this.dob = dob;
+        this.description = description;
+        this.socialLinks = socialLinks;
+        this.sensitiveWords = sensitiveWords;
+        this.interests = interests;
+        this.groups = groups;
+        this.friends = friends;
+        this.blockedUsers = blockedUsers;
+
+        this.creationTime = creationTime;
+    }
+
+    public String getUserName() {
+        return this.userName;
+    }
+
+    public String getProfileName() {
+        return this.profileName;
+    }
+    public void setProfileName(String profileName) {
+        this.profileName = profileName;
+    }
+
+    public LocalDate getDob() {
+        return this.dob;
+    }
+    public void setDob(LocalDate dob) {
+        this.dob = dob;
+    }
+
+    public String getDescription() {
+        return this.description;
+    }
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public List<String> getSocialLinks() {
+        return this.socialLinks;
+    }
+    public void setSocialLinks(List<String> socialLinks) {
+        this.socialLinks = socialLinks;
+    }
+    public void addSocialLinks(String socialLink) {
+        this.socialLinks.add(socialLink);
+    }
+    public void removeSocialLinks(String socialLink) {
+        this.socialLinks.remove(socialLink);
+    }
+
+    public List<String> getSensitiveWords() {
+        return this.sensitiveWords;
+    }
```

Comment:
This is just a suggestion. It seems that you have multiple "nouns" that have the same functions of adding and removing them. Maybe you can try to create different classes i.e. social links, descriptions Dob, and let them share the same interface that can add and remove things.